front/app/components/admin/PostManager/components/FilterSidebar/statuses/StatusFilters.tsx

```diff
@@ -71,14 +72,23 @@ const StatusFilters = ({
           </Button>
         </Box>
       )}
-      {statuses.map((status) => (
-        <StatusFilter
-          key={status.id}
-          status={status}
-          active={isActive(status.id)}
-          onClick={() => handleItemClick(status.id)}
-        />
-      ))}
+      {statuses.map((status) =>
+        status.attributes.code === 'prescreening' ? (
+          <ScreeningStatusFilter
+            key={status.id}
+            status={status}
```

Comment:
I guess the status also doesn't need to be a prop anymore?

Reply:
We are still using it.